can_messages.asciipb

```diff
@@ -24,8 +24,20 @@ msg {
 msg {
   id: 1
   source: CHAOS
+  # target: Driver Controls
+  msg_name: "power distribution fault"
```

Comment:
Do we not have a BPS fault message?

We could just use 1 fault message and code the reason in the data field.

Reply:
Wait, how does that work? My understanding was that BPS should have an explicit fault message so driver controls/lights can pick up the fault. Does this require adding watchdogs to lights/driver controls that expect a valid heartbeat message from BPS?

Reply:
My understanding was that Light and Driver controls would listen to but not active ACK the BPS heartbeat message. If we wanted they can also actively ACK the message; however, they wouldn't require watchdogs as Power Distribution already supplies this functionality?

I recall discussing this with both yourself and Karl.

Reply:
Unless power distribution rebroadcasts the BPS fault in the case of a watchdog timeout, they would need to have their own watchdogs. Otherwise, if Plutus somehow dies without broadcasting a heartbeat message, they wouldn't fault.

We should probably document the fault behavior on Confluence/update any existing documentation. Obviously it's been way too long since this was discussed. :)

Reply:
Ack. We can migrate the `bps_heartbeat` module from Chaos into `ms-helper` in that case. I'll just make it slightly more flexible.